openmc/universe.py

```diff
@@ -701,13 +705,24 @@ def get_unique_universes(self):
 
         """
 
-        unique_universes = np.unique(self._universes.ravel())
-        universes = {}
-
-        for universe in unique_universes:
-            universes[universe._id] = universe
-
-        return universes
+        univs = dict()
```

Comment:
Unfortunately the compiler package is [gone in Python 3](http://stackoverflow.com/questions/16176742/python-3-replacement-for-deprecated-compiler-ast-flatten-function).